slices/erlang-base.yaml

```diff
@@ -0,0 +1,44 @@
+package: erlang-base
+
+essential:
+  - erlang-base_copyright
+
+slices:
+  bins:
+    essential:
+      - bash_bins
+      - coreutils_bins
+      - erlang-base_modules
+      - libc6_libs
+      - libgcc-s1_libs
+      - libreadline8t64_libs
```

Comment:
- bash is needed because there are several erlang packages that have shell scripts, and they all depend on erlang-base, so it makes sense to put bash as dependency in erlang-base.
- coreutils is needed because some bash scripts use `/usr/bin/env` to detect the path of `escript`, and `env` is found in coreutils
- as for libtinfo8t64, I found that some of the binaries in erlnag-snmp are trying to open libtinfo so file (from strace output), and I am not sure if there are others that use it as well, so I put it in erlang-base's essential as a precaution.